src/plugins/intel_gna/gna_model_serial.cpp

```diff
@@ -528,7 +530,7 @@ void GNAModelSerial::Export(void * basePointer, size_t gnaGraphSize, std::ostrea
     }
 
     // once structure has been written lets push gna graph
-    os.write(reinterpret_cast<char*>(basePointer), gnaGraphSize);
+    os.write(reinterpret_cast<char*>(basePointer), ro_size);
```

Comment:
rw_size?

Reply:
No, this is right, gnaGraphSize = ro_size + rw_size, we removed rw region at all.